src/Models/Scopes/Product/WithStockQtyScope.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Rapidez\Msi\Models\Scopes\Product;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
+use Rapidez\Msi\Models\Scopes\Product\WithProductStockScopeMsi;
+
+class WithStockQtyScope extends WithProductStockScopeMsi
+{
+    public function apply(Builder $builder, Model $model)
+    {
+        $stockId = config()->has('rapidez.stock_id') ? config('rapidez.stock_id') : $this->getInventoryStockId();
```

Comment:
same

Reply:
Changed to use fallback from config funtion itself.